db/hints/sync_point.hh

```diff
@@ -57,5 +58,23 @@ struct sync_point_v1 {
     db::hints::per_manager_sync_point_v1 mv_sp;
 };
 
+// IDL type
+struct per_manager_sync_point_v2 {
```

Comment:
It's a bit confusing that it's called `v2` while it is used for the `v3` format. Perhaps we should rename IDL structs ending with `_v1` to end with `_v1_or_v2` and then introduce structures for `v3`? It should be easier to understand that way.

Reply:
done